src/extension.ts

```diff
@@ -95,6 +95,12 @@ export function activate(context: ExtensionContext) {
         languageClient.sendRequest(ExecuteCommandRequest.type, request);
     }));
 
+    languageClient.onReady().then(() => {
+        languageClient.onNotification('vscode/viewProblems', () => {
```

Comment:
we should maybe make this an explicitly sourcery namespace to avoid clashes - sourcery_vscode/viewProblems?

Reply:
How about `sourcery/vscode/viewProblems`?